tools/gethostlatency.bt

```diff
@@ -31,7 +31,10 @@
  * 08-Sep-2018	Brendan Gregg	Created this.
  */
 
-config = { missing_probes = "ignore" }
+config = {
+	missing_probes = "ignore";
```

Comment:
I guess with this it shouldn't matter if some platforms don't have the added (or older) uprobes.